templates/apps/sumo-aws-apps/security-hub-org/security-hub-org-main.yaml

```diff
@@ -222,71 +201,31 @@ Parameters:
     MinLength: 12
     MaxLength: 12
 
-  EnableCISStandard:
+  SecurityStandards:
     Type: String
-    Description: CIS AWS Foundations Standard
-    Default: true
-    AllowedValues:
-      - true
-      - false
-
-  CISStandardVersion:
-    Type: String
-    Description: CIS Standard Version
-    Default: '1.2.0'
-
-  EnablePCIStandard:
-    Type: String
-    Description: Payment Card Industry Data Security Standard (PCI DSS)
-    Default: true
-    AllowedValues:
-      - true
-      - false
-
-  PCIStandardVersion:
-    Type: String
-    Description: PCI Standard Version
-    Default: '3.2.1'
-
-  EnableSBPStandard:
-    Type: String
-    Description: Security Best Practices Standard
-    Default: true
-    AllowedValues:
-      - true
-      - false
-
-  SBPStandardVersion:
-    Type: String
-    Description: Security Best Practices Standard Version
-    Default: '1.0.0'
-
-  RegionsToEnable:
-    Type: String
-    Description: Comma delimited list of regions to enable Security Hub.
-    Default: us-west-1
+    Description: 
+      "AWS Security Hub currently supports the security standards detailed in this section. 
+      Choose a standard to view more details about it and the controls that apply to it. Comma delimited list of standards (e.g. FSBP_v1.0.0,CIS_v1.2.0,PCI_DSS_v3.2.1,NIST_v5.0.0,CIS_v1.4.0)
+      Supports: FSBP_v1.0.0 (AWS Foundational Security Best Practices v1.0.0 standard), CIS_v1.2.0 (Center for Internet Security (CIS) AWS Foundations Benchmark v1.2.0 standard),
+      CIS_v1.4.0 (Center for Internet Security (CIS) AWS Foundations Benchmark v1.4.0 standard), NIST_v5.0.0 (National Institute of Standards and Technology SP 800-53 Rev. 5)"
+    Default: "FSBP_v1.0.0,CIS_v1.2.0,PCI_DSS_v3.2.1"
 
   RegionLinkingMode:
-    AllowedValues: [SPECIFIED_REGIONS, ALL_REGIONS]
+    AllowedValues: [SPECIFIED_REGIONS, ALL_REGIONS, ALL_REGIONS_EXCEPT_SPECIFIED ]
     Default: SPECIFIED_REGIONS
     Description:
-      Indicates whether to aggregate findings from all of the available Regions in the current partition. Also determines whether to automatically
+      "Indicates whether to aggregate findings from all of the available Regions in the current partition. Also determines whether to automatically
       aggregate findings from new Regions as Security Hub supports them and you opt into them.
+      ALL_REGIONS – Security Hub aggregates data from all Regions. Security Hub also aggregates data from new Regions as they are supported and you opt into them.
+      ALL_REGIONS_EXCEPT_SPECIFIED – Security Hub aggregates data from all Regions except for Regions that you want to exclude. Security Hub also aggregates data from new Regions as they are supported and you opt into them. Use Regions to provide the list of Regions to exclude from aggregation.
```

Comment:
If one selects All REGIONS and LinkedRegions contains only one region will it still collect from all regions ?
Also if one selects ALL_REGIONS_EXCEPT_SPECIFIED where he will specified the excluded regions in linkedregions parameter ?

Reply:
1. i will test.
2. input the excluded regions, it is LinkedRegions
3. i will add SecurityHubHomeRegion, and use input LinkedRegions

Reply:
1. fix error